.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -26,27 +28,37 @@ assignees: ''
 - [ ] Add versioned documentation build on docs.seqan.de.
   <details><summary>Click for detailed steps</summary>
 
-  1. Build the documentation locally
+  1. Build the documentation locally or use the artefact generated by the CI
 
   2. Create a #.#.# directory for the release in `/web/docs.seqan.de/htdocs/seqan/`
 
-  3. Copy everything from the local build (`doc_usr/html/*`) into the directory.
+  3. Copy everything from the build (`doc_usr/html/*`) into the directory.
 
-  4. Alter the file `/web/docs.seqan.de/htdocs/index.html` with a link to the new documentation build.
+  4. Alter the file `/web/docs.seqan.de/htdocs/seqan3.html` with a link to the new documentation build.
 
   </details>
-- [ ] Prepare `seqan-[VERSION]-with-submodules.tar.gz`.
+- [ ] Prepare `seqan3-[VERSION]-{Linux,Source}.tar.xz{,.sha265}`.
   <details><summary>Click for detailed steps</summary><br>
 
   Make a fresh clone of the repository and recursively delete the `.git` directories.
```

Comment:
What do we need to delete now? Only the main `.git` from the checkout?

Reply:
Nothing needs to be delete